idb/postgres/postgres_migrations.go

```diff
@@ -34,6 +34,7 @@ func init() {
 		{m3acfgFix, false, "Recompute asset configurations with corrected merge function."},
 		{m4RewardsAndDatesPart1, true, "Update DB Schema for cumulative account reward support and creation dates."},
 		{m5RewardsAndDatesPart2, false, "Compute cumulative account rewards for all accounts."},
+		{m6MarkTxnJSONSplit, true, "record round at which txn json recording changes, for future migration to fixup prior records"},
```

Comment:
This will cause `m5RewardsAndDatesPart2` to block, so it will need to be moved to m4.

Reply:
m5 has been released as 2.3.0, so I don't think we can move it.
m5 only becomes blocking if both are applied at the same time. If an Indexer has been upgraded to 2.3.0, then this (call it 2.3.1 ?) should be fine.